README.md

```diff
@@ -124,8 +126,8 @@ We use GitHub Secrets to store sensitive environment variables. A template `.env
 The file is organized into three main sections:
 
 - **Postgres Environment Variables**. This section contains the credentials to connect to the PostgreSQL database, such as the username, password, and the name of the database.
-- **Backend Environment Variables**. These variables are used by the backend (e.g., FastAPI) to configure its behavior and to connect to the database and the frontend application.
-- **Frontend Environment Variables**. This section contains the base URL for API calls to the backend and `NODE_ENV` variable that determines in which environment the Node.js application is running.
+- **Backend Environment Variables**. These variables are used by the backend (i.e., FastAPI) to configure its behavior and to connect to the database and the frontend application.
```

Comment:
I have no idea how this got changed tbh. Is there some crazy linter that edits abbreviations?